### ISSUE TITLE
build problem reported by Brian Toby

### ISSUE THREAD

**Issue by sbillinge** [diffpy/diffpy.pdfgui#73]
Hi Simon

  I recently tried building a new copy of PDFfit2 for Python 3.9 and came up with a problem via “pip install". The build requires the GSL (Gnu Scientific Library) but the setup.py does not seem to test for that. Once I installed it manually, the build completed. I have not tested it yet, but that will come soon. In any case the build error is below.

Brian

See stack trace in hidden text

**Comment by bobleesj** [comment]
> Python 3.9 and came up with a problem via “pip install" 

@bobleesj Double check this after re-cookiecugting and proper pdffit2 version.

**Comment by sbillinge** [comment]
Can we close this? I think it is done but it's mistakenly here in PDFgui and not in pdffit2 where the fix was done of I am right. 

To fully close it let's get sign of from Brian that it is working (we may already have that sign of from the pdffit2 merge.  

Tl;Dr, let's get this to closed

**Comment by sbillinge** [comment]
This should be fixed by the most recent pdffit2 release. Move this to full-release and ask Brian to test in on the pre-relase, then close

**Comment by sbillinge** [comment]
I responed to Brian by email to test with the latest versions of the code today.

**Comment by sbillinge** [comment]
closing as fixed
